cmake/SearchForStuff.cmake

```diff
@@ -164,9 +164,12 @@ if (PKG_CONFIG_FOUND)
   # Find Simbody
   set(SimTK_INSTALL_DIR ${SimTK_INSTALL_PREFIX})
   #list(APPEND CMAKE_MODULE_PATH ${SimTK_INSTALL_PREFIX}/share/cmake)
-  find_package(Simbody)
-  if (Simbody_FOUND)
-    message (STATUS "Looking for Simbody - found")
+  find_package(Simbody CONFIG)
+  if (Simbody_FOUND)  
+    if ("${Simbody_LIBRARIES}" STREQUAL "" OR "${Simbody_LIBRARIES}" STREQUAL "Simbody_LIBRARIES-NOTFOUND")
+      set(Simbody_LIBRARIES SimTKmath SimTKcommon SimTKsimbody)
+    endif()
+    message (STATUS "Simbody_LIBRARIES: ${Simbody_LIBRARIES}")
```

Comment:
This seems a debug message, can we remove it?
```suggestion
```

Reply:
done.